### PR TITLE
Initialised basic Angular project

### PR DESCRIPTION
**Pull request by pschildkamp**
To bootstrap this project I've added the basic scaffolding to start building the LITTIL frontend website.
- Initialised a default Angular project
- Created an docker image based on a Nginx
- Added documentation on how to start when contributing to LITTIL

### CODE REVIEW

File: docs/set-up-your-development-environment.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+### Nodejs
+Nodejs is used to generate the static files via Angular. The LITTIL website is tested and build on the LTS version of NodeJS 16.
+When installing NodeJS you will get an installation of NPM as well. 
+
+#### Windows & MacOS
+To install NodeJS on Windows or MacOS you can simply download it from [the website](https://nodejs.org/en/).
+
+#### Linux
+The easiest way to install NodeJS is by using your package manager. For instructions on your distribution see [the website asell](https://nodejs.org/en/download/package-manager/)
```
Comment (LDMGN) [inline docs/set-up-your-development-environment.md:16]: Typo? (asell)

**Review by LDMGN — APPROVED** [review]
Looks pretty complete (Angular, Docker, documentation) to me. Nice practice to have separate build and host containers!

Next to reading through the files, I checked out the project, installed dependencies, and ran it succesfully. Also built a docker image and ran that succesfully (building the Dockerfile.build and then running the image produced).

File: README.md [hunk 2]
```diff
@@ -1 +1,30 @@
-# littil-frontend
\ No newline at end of file
+# LITTIL website
+
+This project contains the sourcecode of the LITTIL website.
+
+## Setup your local environment
+For more information about setting up your local development environment, please read [set up your development environment](https://github.com/Devoxx4Kids-NPO/littil-frontend/blob/main/docs/set-up-your-development-environment.md).
```
Comment (jaspersprengers) [inline README.md:6]: link doesn't seem to work.

Comment (pschildkamp) [reply README.md:6]: I've added a link referring to the main branch, which doesn't exist yet. If there is a way to refer to an md file in the repository without using absolute URL's then please let me know so I can fix this.

LTDR; the URL should work when this PR is merged to the main branch.

File: .vscode/tasks.json [hunk 3]
```diff
@@ -0,0 +1,42 @@
+{ime to stay on this plan when your trial ends.
```
Comment (Stefanic) [inline .vscode/tasks.json:1]: Broken line

File: docs/set-up-your-development-environment.md (hunk 1 above)
Comment (pschildkamp) [reply docs/set-up-your-development-environment.md]: Fixed.

File: README.md (hunk 2 above)
Comment (pschildkamp) [reply README.md:6]: Apparently you can use relative paths on Github. Fixed the URI. It works now.

File: .vscode/tasks.json (hunk 3 above)
Comment (pschildkamp) [reply .vscode/tasks.json]: I removed all the comments in the generated JSON files. It's not conform JSON standards anyway.

**Pull request merged**